rst2pdf/extensions/plantuml_r2p.py

```diff
@@ -92,5 +92,4 @@ def gather_elements(self, client, node, style):
         # Add Image node with the right image
         return [MyImage(tfile.name, client=client)]
 
-
```

Comment:
This was accidental, I assume?

Reply:
I think it just looked wrong to me, I thought I was tidying up! I can remove it if it's going to cause a problem

Reply:
Yeah, the `lint` job will fail with this because it's not PEP-8 compliant. You need to re-add this for it to merge.

Reply:
got it! I had a lot of debug in there and didn't realise I hadn't quite reset it all. The build did fail so it should have been obvious the PR needed work.